adage/analyze/api.py

```diff
@@ -5,7 +5,7 @@
 from organisms.api import OrganismResource
 from genes.api import GeneResource
 from tastypie import fields, http
-from tastypie.resources import Resource, ModelResource
+from tastypie.resources import Resource, ModelResource, convert_post_to_VERB
```

Comment:
Sure. I got a few basic tests written up yesterday. I will be sure to include the case you describe, too.